app/views/component-detail.html

```diff
@@ -36,6 +36,25 @@
 						<p><a href="{{service_url}}" class="btn btn-lg btn-primary active"><i class="glyphicon glyphicon-circle-arrow-right"></i> Load service</a> &nbsp; (services host their own documentation)</p>
 					{% endif %}
 
+					{% if origami_type == 'imageset' %}
+						<h2>Quick start</h2>
+
+						<h3>Use the imageset</h3>
+						<p>Imageset images can be loaded directly using the imageservice. This shortcode for this imageset is: <code>{{imageset_scheme}}</code></p>
+
+						<p>To load an image from this imageset, use the following URL for the build service:</p>
```

Comment:
To load an image from this imageset use the following URL from the Image Service: